fixtures/_integration.cpp.sem.uast

```diff
@@ -208,6 +176,27 @@
          ],
       },
    ],
-   'Prop_IncludeDirectives': [],
+   'Prop_PreprocStatements': [
+      { '@type': "uast:InlineImport",
+         '@pos': { '@type': "uast:Positions",
+            start: { '@type': "uast:Position",
+               offset: 0,
+               line: 1,
+               col: 1,
+            },
+            end: { '@type': "uast:Position",
+               offset: 19,
+               line: 1,
+               col: 20,
+            },
+         },
+         All: true,
+         Names: [],
+         Path: { '@type': "uast:Identifier",
```

Comment:
It's probably `uast:String`, not `uast:Identifier` (since it's a path).

Reply:
Would be a little odd to tag it as a string when there aren't quotes (`<iostream>`).

Reply:
By the definition, Identifiers are names of objects. `iostream` is more a filename/path and other drivers express it as a `String` instead of `Identifier`. As mentioned above, there is a different way to represent `<` (`IsSystem`).

Reply:
Done in next commit.